packages/shopify-cart/src/types/shopify.type.ts

```diff
@@ -5460,12 +5660,12 @@ export type ProductVariant = HasMetafields & Node & {
   availableForSale: Scalars['Boolean'];
   /** The barcode (for example, ISBN, UPC, or GTIN) associated with the variant. */
   barcode?: Maybe<Scalars['String']>;
+  /** The compare at price of the variant. This can be used to mark a variant as on sale, when `compareAtPrice` is higher than `price`. */
+  compareAtPrice?: Maybe<MoneyV2>;
```

Comment:
Looks like we're using a few of the PriceV2 and associated fields in the [merchandise fragment](https://github.com/getnacelle/nacelle-js/blob/95977b55a7faa099a662781c3327220b6d0e824e/packages/shopify-cart/src/graphql/fragments/merchandise.ts#L4)

Reply:
Good catch! Thanks. Fixed in https://github.com/getnacelle/nacelle-js/pull/280/commits/097927b515357ba577e47d478d1f3ad9e3030855.